exonum/src/blockchain/block.rs

```diff
@@ -60,6 +107,49 @@ pub struct Block {
     /// These calls can include transactions, `before_transactions` and/or `after_transactions` hooks
     /// for services.
     pub error_hash: Hash,
+    /// Some additional entries that can be added into the block.
```

Comment:
"Some additional entries" sounds weird. I'd rephrase it like "Additional block information".